### PR TITLE
fix: fix CI fails

### PR DESCRIPTION
**Pull request by ricolin**
fix https://github.com/vexxhost/atmosphere/issues/1033

### CODE REVIEW

**Comment by okozachenko1203** [comment]
@ricolin  i can see you did a fix for ovn image build.
Btw, i think this will happen in the future again because we use head commit of upstream ovn-kubernetes project.
I proposed a permanent fix for this https://github.com/vexxhost/atmosphere/pull/1030

**Comment by okozachenko1203** [comment]
@ricolin  https://github.com/vexxhost/atmosphere/issues/1033
I opened a new ticket for ovn latest change. Could you pin the commit hash?
Thanks.

File: Jenkinsfile [hunk 1]
```diff
@@ -142,15 +142,36 @@ pipeline {
...
 
         stages {
+          stage('keycloak') {
+            when { expression { env.SCENARIO == "keycloak" } }
+            steps {
+              // Checkout code with built/pinned images
+              unstash 'src-with-pinned-images'
+
+              // Install dependencies
+              sh 'sudo apt-get install -y git python3-pip docker.io'
```
Comment (ricolin) [inline Jenkinsfile:160]: not sure if we have better way to introduce docker to this job

**Comment by ricolin** [comment]
TODO: I will take follow up task to move keycloak job out of container like aio, so things can go faster and we can use current fetch logs script directly to get logs archieve

**Comment by ricolin** [comment]
replace with https://github.com/vexxhost/atmosphere/pull/1049

**Pull request closed without merging**